apps/web/components/ListViewCard/ListViewCard.tsx

```diff
@@ -125,21 +125,27 @@ export const ListViewCard = ({
           </Box>
         )}
         <Box paddingTop={3} width="full" height="full">
-          <Box display={'flex'} alignItems={'flexEnd'} height="full">
+          <Box
+            display={'flex'}
+            alignItems={'flexEnd'}
+            height="full"
+            width="full"
+            style={{ cursor: cta.disabled ? 'not-allowed' : '' }}
```

Comment:
It's probably harmless to set the cursor to an empty string but maybe it's safer to set it to undefined so it behaves the default way

Reply:
Agree, fixed